src/containers/tutor-home-page/add-photo-step/AddPhotoStep.jsx

```diff
@@ -61,7 +79,18 @@ const AddPhotoStep = ({ btnsBox, stepLabel }) => {
             validationData={ validationData }
           />
         </Box>
-
+        { (isMobile || isTablet ) && (
```

Comment:
As i see, it is the same like above, you can move this box to variable and use it in different screen sizes